client/dist/styles/questions-and-answers.css

```diff
@@ -0,0 +1,97 @@
+.q-and-a-form {
+  font-size: 9pt;
```

Comment:
Was the pt unit intentional or did you want it to be px? It looks like pt is a valid unit though. I've just never seen it